src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -7348,7 +7348,7 @@ private BoundExpression BindArrayAccess(ExpressionSyntax node, BoundExpression e
                 : new BoundArrayAccess(node, expr, convertedArguments.AsImmutableOrNull(), resultType, hasErrors: false);
         }
 
-        private BoundExpression ConvertToArrayIndex(BoundExpression index, SyntaxNode node, DiagnosticBag diagnostics, bool allowIndexAndRange)
+        private BoundExpression ConvertToArrayIndex(BoundExpression index, SyntaxNode node, DiagnosticBag diagnostics, bool allowIndexAndRange, bool allowNIntAndNUInt)
```

Comment:
Perhaps name the parameter `allowNativeIntegers`.

Reply:
Renamed